influxql/ast.go

```diff
@@ -320,6 +320,18 @@ type CreateDatabaseStatement struct {
 	// IfNotExists indicates whether to return without error if the database
 	// already exists.
 	IfNotExists bool
+
+	// RetentionPolicyExists indicates whether the user explicitly wants to create a retention policy
```

Comment:
Minor, but I don't like this variable name. How about something like `RetentionPolicyCreate`?